templates/watcher/config/00-default.conf

```diff
@@ -0,0 +1,43 @@
+[DEFAULT]
+state_path = /var/lib/watcher
+transport_url = {{ .TransportURL }}
+control_exchange = watcher
+debug = True
+
+[database]
+connection = {{ .DatabaseConnection }}
+
+[oslo_policy]
+policy_file = /etc/watcher/policy.yaml.sample
+
+[oslo_messaging_notifications]
+driver = messagingv2
+
+[keystone_authtoken]
+memcached_servers = {{ .MemcachedServers  }}
+# TODO jgilaber implement handling this option when we add tls support
+# cafile = /var/lib/ca-bundle.pem
+project_domain_name = Default
+project_name = service
+user_domain_name = Default
+password = {{ .ServicePassword  }}
+username = {{ .ServiceUser  }}
+auth_url = {{ .KeystoneAuthURL }}
+interface = internal
+auth_type = password
+
+[watcher_clients_auth]
+memcached_servers = {{ .MemcachedServers  }}
+# TODO jgilaber implement handling this option when we add tls support
+# cafile = /var/lib/ca-bundle.pem
+project_domain_name = Default
+project_name = service
+user_domain_name = Default
+password = {{ .ServicePassword  }}
+username = {{ .ServiceUser  }}
+auth_url = {{ .KeystoneAuthURL }}
+interface = internal
+auth_type = password
+
+[oslo_concurrency]
+lock_path = /var/lib/watcher/tmp
```

Comment:
In puppet-watcher deployment, i don't find the memcached_servers option under watcher_clients_auth. However, I see another section [cache]:

[cache]
backend=dogpile.cache.pymemcache
enabled=True
memcache_servers=<server list>
tls_enabled=False

You can see a config file documented with all the available options in:

https://logserver.rdoproject.org/openstack-periodic-daily/review.rdoproject.org/rdoinfo/master/weirdo-epoxy-promote-puppet-scenario001-centos-stream-9/1d7b114/logs/weirdo-project/logs/etc/watcher/watcher.conf.txt.gz

Reply:
In the documented file I see both the `memcached_servers` option under the `keystone_authtoken` section and `memcache_servers` under the `cache` section. I'm not really sure if only one of the two or both should be set

Reply:
I removed the one from `watcher_clients_auth` though

Reply:
I'd keep it in both [keystone_authtoken] and [cache] sections